classes/ElasticPressHooks.php

```diff
@@ -65,4 +69,66 @@ public function filterMimeTypes($mime_types): array
 
         return $mime_types;
     }
+
+    public function preventDashboardIndex($index_meta)
+    {
+        // TODO: prevent dashboard sync from ElasticPress UI
+
+        return $index_meta;
+    }
+
```

Comment:
Hey dude, this is effectively done by defining a constant.
define('EP_DASHBOARD_SYNC', false);

Reply:
I thought this was a function you added in..sounds like you since removed it, maybe this is left over from an old branch, I'll remove it.